### DISCUSSION TITLE
[Suggestion] Add ability request auctions from the API by ID

### DISCUSSION THREAD

**Discussion by skyeredwood** [HypixelDev/PublicAPI#441] [General]
The name pretty much says it all. When requesting auctions from the /skyblock/api route an "_id" property is returned which could be used by developers to request specific auctions. This would be incredibly useful to help, especially for applications or APIs which return specific auctions based on their ID.

**Comment by ConnorLinfoot** [comment]
You can already request an specific auction using the `/skyblock/auction` endpoint as seen here https://api.hypixel.net/#tag/SkyBlock/paths/~1skyblock~1auction/get

**Reply by skyeredwood** [reply]
Yes, you can request it by the player's ID but not by the specific ID of the auction itself; you can only request by the player's UUID, profile or username. (also sorry for posting this in issues that was a misclick)

**Reply by ConnorLinfoot** [reply]
The UUID is the auction, not the player. There is no support for requesting by player name, only player UUID.

`player` argument is the player UUID.
`profile` argument is the profile UUID.
`uuid` argument is the auction UUID.

We'll tweak the documentation to make this slightly more clear, as a lot of users likely believe UUID alone relates to a player.

**Reply by skyeredwood** [reply]
Thanks so much :)
